### DISCUSSION TITLE
How to Contribute

### DISCUSSION THREAD

**Discussion by gadorlhiac** [lcls-users/lcls-users#20] [Ideas]
We should consider show to standardize submission guidelines and streamline the contribution process. Some thoughts:

- A standard template with consistent headers for a boilerplate framework
- Simply a list of guidelines in the readme and associated homepage
- Ways of auto-linking other documentation sources
- Perhaps a reminder system (Yearly? Not sure what timeframe makes sense) to notify us when pages should be reviewed to avoid stale information

**Comment by fredericpoitevin** [comment]
- separate merging to main and adding to navigation steps using template PR.
